corehq/apps/cleanup/models.py

```diff
@@ -10,3 +10,17 @@ class Meta:
         constraints = [
             models.UniqueConstraint(fields=['doc_id', 'doc_type'], name='deletedcouchdoc_unique_id_and_type')
         ]
+
+
+class DeletedSQLDoc(models.Model):
+    doc_id = models.CharField(max_length=126)
+    doc_type = models.CharField(max_length=255)
```

Comment:
`doc_type` is probably fine, but I'm thinking of how [django-field-audit](https://github.com/dimagi/django-field-audit/blob/main/field_audit/models.py#L205) stores class information and makes it clear that the full class path should be saved (it is also less couch-centric).

It isn't clear to me here if we expect to only save the class name or the full path. The full path is useful since you can use the path dynamically in code and it helps to avoid conflicts (might never actually impact us), but again maybe the intention is to store the full class path already, in which case I think a clearer name or comment would be helpful.

Reply:
Yea that makes sense/would make it clearer. Just making sure but when you say full class path you mean the `app_name.ModelName` format right?